src/EventStore.Core/ExtTcpOptions.cs

```diff
@@ -1,39 +1,6 @@
-﻿using System;
-
-namespace EventStore.Core;
+﻿namespace EventStore.Core;
 
 public class ExtTcpOptions {
```

Comment:
```suggestion
// Still needed for gossip
public class ExtTcpOptions {
```

Reply:
It we want to be thorough, it's also used to gather tcp stats too